canonical-web-and-design/maas-ui
# refactor(all): remove all references to general.navigation_options websocket method (#2683)

## Done

- Backport of #2683 

## QA steps

- Run the full UI pointing at bolla and check that the websocket stays connected
- Go to a React page and check that `general.navigationOptions` does not exist in the redux store and no action is dispatched to fetch them
- Go to a legacy page and check that no websocket call is made to `general.navigation_options`
- Check that legacy pages still render properly
